mongodb_store/scripts/replicator_node.py

```diff
@@ -139,9 +148,21 @@ def do_dump(self, collection, master, less_time_time=None, db='message_store'):
 
         # print args
 
-        subprocess.call(args)
-
-
+        self.dump_process = subprocess.Popen(args)
+        self.dump_process.wait()
+
+
+    def do_cancel(self):
+        if self.restore_process and self.restore_process.poll() is None:
```

Comment:
@hawesie Thank you for review.
> Is this to check if these are not None?
No, I did just for binding subprocess instance to the variable `self.dump_process` or `self.restore_process` in order to send cancel signal in `do_cancel` method
https://github.com/strands-project/mongodb_store/pull/144/files#diff-dc4579a0ffb640360f905236a878452fR161